scripts/delete_old_spaces.py

```diff
@@ -0,0 +1,47 @@
+import argparse
+import datetime
+from typing import Optional
+
+from huggingface_hub import HfApi
+
+
+def delete_space(space_id: str, hf_token: str, api_client: Optional[HfApi] = None):
+    api_client = api_client or HfApi()
+    api_client.delete_repo(repo_id=space_id, token=hf_token, repo_type="space")
+
+
+def get_spaces_to_delete(
+    n_days: int, org_name: str, api_client: Optional[HfApi] = None
+):
+    api_client = api_client or HfApi()
```

Comment:
Out of curiosity, why pass in an `api_client` as a parameter like this? We can just define a single instance of `HfApi()` outside of the function?

Reply:
I think it makes the function more general in the event we want to reuse them in other scripts in the future. We do currently define a single instance of api_client and pass it to all the functions.